src/main/java/frc/robot/subsystems/intake/Intake.java

```diff
@@ -0,0 +1,62 @@
+package frc.robot.subsystems.intake;
+
+import com.ctre.phoenix.motorcontrol.ControlMode;
+import com.ctre.phoenix.motorcontrol.can.TalonSRX;
+import edu.wpi.first.wpilibj.Solenoid;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+
+/**
+ * Add&named motors in Intake.
+ */
+public class Intake extends SubsystemBase {
+    private TalonSRX motor = new TalonSRX(Ports.Intake.MOTOR);
+    private Solenoid piston = new Solenoid(Ports.Intake.SOLENOID);
+
+    /**
+     * Add motor inverted.
+     */
+    public void TalonSRXSelenoid() {
+        this.motor.setInverted(Ports.Intake.IS_MOTOR_INVERTED);
+    }
+
+    /**
+     * Set power for motor.
+     * @param power the speed times the torque.
+     */
+    public void setMotorPower(double power) {
```

Comment:
Rename to setPower

Reply:
I disagree, it might not be the proper naming, maybe `powerWheels(...)`?

Reply:
Sure it's just repetitive to call the method setMotorPower